google/cloud/pubsub/samples/samples.cc

```diff
@@ -1449,6 +1516,10 @@ void AutoRun(std::vector<std::string> const& argv) {
   PublishHelper(publisher, "SubscriberRetrySettings", 1);
   SubscriberRetrySettings({project_id, subscription_id});
 
+  std::cout << "\nRunning OptomisticSubscribe() sample" << std::endl;
+  PublishHelper(publisher, "OptomisticSubscribe", 1);
+  SubscriberRetrySettings({project_id, topic_id, subscription_id});
```

Comment:
s/Opto/Opti/

And it should call `OptimisticSubscribe()`, not `SubscriberRetrySettings()`

Reply:
My bad. Thanks.